CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 CHANGELOG
 =========
 
+5.1.0 (2023-08-10)
```

Comment:
Did we release it 08-10? Normally we don't set a date in the changelog until we release.

Reply:
Oh, I'll remove it. Thank you!